core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/resolution/ResolutionModifier.java

```diff
@@ -34,22 +38,21 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ResolutionModifier.class);
 
-  private static final String PARAM_MAXIMIZE = "maximize";
-
   private static final String WIDTH_PARAM = "width";
 
   private static final String HEIGHT_PARAM = "height";
 
-  private static final int MAX_SIZE = 100000;
+  private static final String JAVASCRIPT_GET_BODY_HEIGHT = "return document.body.scrollHeight";
```

Comment:
Please add those changes into resolution modifier documentation:
- height param is no longer mandatory,
- max page height,
- how does AET detect page height,
- remove `maximize` parameter

Reply:
Updated